extensions/tfchain/consensus/validators.go

```diff
@@ -0,0 +1,162 @@
+package consensus
+
+import (
+	"fmt"
+
+	"github.com/threefoldfoundation/tfchain/pkg/config"
+	"github.com/threefoldtech/rivine/modules"
+	"github.com/threefoldtech/rivine/modules/consensus"
+	"github.com/threefoldtech/rivine/types"
+)
+
+func GetStandardTransactionValidators() []modules.TransactionValidationFunction {
+	return []modules.TransactionValidationFunction{
+		consensus.ValidateTransactionFitsInABlock,
+		consensus.ValidateTransactionArbitraryData,
+		consensus.ValidateCoinInputsAreValid,
+		consensus.ValidateCoinOutputsAreValid,
+		consensus.ValidateBlockStakeInputsAreValid,
+		consensus.ValidateBlockStakeOutputsAreValid,
+		consensus.ValidateDoubleCoinSpends,
+		consensus.ValidateDoubleBlockStakeSpends,
+		consensus.ValidateCoinInputsAreFulfilled,
+		consensus.ValidateBlockStakeInputsAreFulfilled,
+	}
+}
+
+func GetStandardTransactionVersionMappedValidators() map[types.TransactionVersion][]modules.TransactionValidationFunction {
+	const (
+		secondsInOneDay                         = 86400 + config.StandardNetworkBlockFrequency // round up
+		daysFromStartOfBlockchainUntil2ndOfJuly = 74
+		txnFeeCheckBlockHeight                  = daysFromStartOfBlockchainUntil2ndOfJuly *
```

Comment:
this var is wrong, calculates to about block 53354, but there is at least a transaction without miner fee in block 277852, causing the std net sync to fail on that block